x-pack/legacy/plugins/siem/public/store/timeline/helpers.ts

```diff
@@ -102,12 +103,34 @@ export const addTimelineNoteToEvent = ({
   };
 };
 
+interface AddTimelineParams {
+  id: string;
+  timeline: TimelineResult;
+  // timelineById: TimelineById;
+}
+
+/**
+ * Add a saved object timeline to the store
+ * and default the value to what need to be if values are null
+ */
+export const addTimelineToStore = ({ id, timeline }: AddTimelineParams): TimelineById => ({
+  // As right now, We are not managing multiple timeline
+  // for simplification, we do not need the line below
+  // ...timelineById,
```

Comment:
perhaps something like `TODO: revisit this when we support multiple timelines` makes sense here?